README.md

```diff
@@ -0,0 +1,330 @@
+
+# Requirements
+
+## Build:
+
+- **JDK**: Version 8 or newer.
+- **Maven**: Version 3.6.3 or newer.
+
+## Use:
+
+- **JDK**: Version 8 or newer.
+- **Maven**: Version 3.3.9 or newer.
+- **Gradle**: Version 3 or newer.
+
+
+## Adding ShrinkWrap Resolvers to Your Project
+
+To include ShrinkWrap Resolvers in your project, you can add a dependency on the `shrinkwrap-resolver-depchain` module in your Maven `pom.xml`:
+
+```xml
+<dependencies>
+    <dependency>
+      <groupId>org.jboss.shrinkwrap.resolver</groupId>
+      <artifactId>shrinkwrap-resolver-depchain</artifactId>
+      <version>${version.shrinkwrap.resolvers}</version>
+      <scope>test</scope>
+      <type>pom</type>
+    </dependency>
+</dependencies>
+```
+
+If you are using Arquillian BOM prior to version 1.1.0.Final, ensure that you import ShrinkWrap Resolvers BOM preceding Arquillian BOM to get version 2.0.x. Importing the ShrinkWrap BOM is recommended:
+
+```xml
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>org.jboss.shrinkwrap.resolver</groupId>
+      <artifactId>shrinkwrap-resolver-bom</artifactId>
+      <version>${version.shrinkwrap.resolvers}</version>
+      <scope>import</scope>
+      <type>pom</type>
+    </dependency>
+    <!-- Other dependencies -->
+  </dependencies>
+</dependencyManagement>
+```
+
+## Resolving Dependencies
+
+To initiate the resolution of dependencies, utilize the `org.jboss.shrinkwrap.resolver.api.maven.Maven` class. Below are typical scenarios for its usage.
+### Resolution of Artifacts by Maven Coordinates
+
+Maven coordinates follow `groupId:artifactId:[packagingType:[classifier]]:version` format. You can use these coordinates to resolve a file, where `G` is the group ID, `A` is the artifact ID, `P` is the packaging type, `C` is the classifier, and `V` is the version.
+
+#### Resolve a file using Maven coordinates:
+```java
+File[] files = Maven.resolver().resolve("G:A:V").withTransitivity().asFile();
+```
+
+#### Avoid transitive dependencies:
+```java
+File file = Maven.resolver().resolve("G:A:V").withoutTransitivity().asSingleFile();
+```
+
+#### Resolve multiple artifacts:
+```java
+File[] files = Maven.resolver().resolve("G1:A1:V1", "G2:A2:V2").withTransitivity().asFile();
+```
+
+#### Specifying Packaging and Classifier
+
+```java
+File file = Maven.resolver().resolve("G:A:war:V").withTransitivity().asFile();
+File fileWithClassifier = Maven.resolver().resolve("G:A:test-jar:tests:V").withTransitivity().asFile();
+```
+
+#### Returning Resolved Artifacts
+
+ShrinkWrap Resolvers provides various options to retrieve resolved artifacts. You can obtain them as files, streams, URLs, or specific archive types like JavaArchive, WebArchive, or EnterpriseArchive. For example:
+- Get as a URL: `Maven.resolver().resolve("G:A:V").withTransitivity().as(URL.class)`
```

Comment:
I'd replace this with a code block, showing couple of usages, each - perhaps - with a comment explaining the return type ("Get as a URL", "Get as a Java archive", ...)

I'd use URL, JAR, WAR